scapy/main.py

```diff
@@ -258,7 +258,7 @@ def list_contrib(name=None,  # type: Optional[str]
             continue
         if mod.endswith(".py"):
             mod = mod[:-3]
-        desc = {"description": None, "status": None, "name": mod}
+        desc = {"description": "", "status": "", "name": mod}
```

Comment:
The `sort` call on line 284 requires the value to be comparable. On Python3, `None` isn't comparable.